code/iaas/model/src/main/java/io/cattle/platform/core/addon/LoadBalancerCertificate.java

```diff
@@ -0,0 +1,58 @@
+package io.cattle.platform.core.addon;
+
+import io.github.ibuildthecloud.gdapi.annotation.Type;
+
+@Type(list = false)
+public class LoadBalancerCertificate {
```

Comment:
@ibuildthecloud it has one extra field - isDefault. LB can have 1 default certificate + multiple non-default; if no default is specified, "strict SNI" rule will get applied on the backend.